frameworks/cassandra/tests/test_sanity.py

```diff
@@ -24,7 +24,9 @@ def configure_package(configure_security):
             config.PACKAGE_NAME,
             config.get_foldered_service_name(),
             config.DEFAULT_TASK_COUNT,
-            additional_options={"service": {"name": config.get_foldered_service_name()}},
+            additional_options={
```

Comment:
A lot of these changes seem to be related to black formatting. In the config in the project root, we set the line length to 100. So currently:
```
✗ black --diff frameworks/cassandra/tests/test_sanity.py
All done! ✨ 🍰 ✨
1 file left unchanged.
```

Reply:
Oh, I might have missed this. I will revert these changes.